include/boost/decimal/detail/bit_layouts.hpp

```diff
@@ -112,7 +112,7 @@ struct IEEEl2bits
 
 struct IEEEbinary128
 {
-#if BOOST_CHARCONV_ENDIAN_LITTLE_BYTE
+#if (defined(BOOST_CHARCONV_ENDIAN_LITTLE_BYTE) && (BOOST_CHARCONV_ENDIAN_LITTLE_BYTE != 0))
```

Comment:
```suggestion
#if (defined(BOOST_DECIMAL_ENDIAN_LITTLE_BYTE) && (BOOST_DECIMAL_ENDIAN_LITTLE_BYTE != 0))
```

Reply:
In the terse form, if it is not defined at all, then checking its value only leads to warnings (properly so) with certain warnings activated.

Reply:
I meant it as that's a copy paste error. The macro should be BOOST_DECIMAL not BOOST_CHARCONV